tests/plugins/tabletools/manual/insertcolumnverticalsplit.md

```diff
@@ -0,0 +1,8 @@
+@bender-tags: 4.5.7, tc, tabletools, 13729
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, table, tabletools, contextmenu
+
+1. Make a vertical split in the cell 2,2.
+2. Right click in the first cell in 4. column to insert a new column before it.
```

Comment:
Column inserted from cell 4,2 and 4,3 is inserted in the right place (between 3rd and 4th column). However, if you right click in the 4,1 cell and insert column it is placed on the beginning of the table as a first column.

Reply:
It's reproducible also in Safari. Additionally it's reproducible only if the selection is placed at the end of the cell. It seems to be connected with [known issue in Webkit/Blink](https://github.com/ckeditor/ckeditor-dev/pull/653#issuecomment-319969400) (it's nicely visible with `tableselection` plugin enabled; in such case cells 4.1 and 1.2 are selected after inserting new column). Probably very similar workaround as proposed for #577 could solve the issue. However this issue is not connected with the original one, so I'd move it to the dedicated issue.

Reply:
Reported: #900

Reply:
👍